### ISSUE TITLE
Status

### ISSUE THREAD

**Issue by nsaunders** [mechairoi/purescript-unique#2]
Hi there @mechairoi. If you have lost interest in this project, would you please consider allowing me to take over maintainership? Thanks in advance for considering it.

**Comment by mechairoi** [comment]
@nsaunders Sure. How can this be done? Transfer ownership of this repository?

**Comment by nsaunders** [comment]
Thanks @mechairoi, that is wonderful news! If you're OK with it, I would like to transfer the repo to a new namespace, but there are a few steps on my end. For you, however, I think we can keep it pretty simple:

1. Go to [Manage access](https://github.com/mechairoi/purescript-unique/settings/access) in your repository settings.
2. Click on **Add people** and select my username.
3. Assign me the **Admin** role.

If you encounter any unexpected challenges, see [Managing teams and people with access to your repository](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/managing-repository-settings/managing-teams-and-people-with-access-to-your-repository) in GitHub Docs.

Thanks again for helping me to revive this package, and I look forward to getting it working with the latest version of the compiler soon. :-)

**Comment by mechairoi** [comment]
Thank you for the detailed steps. I have sent an invitation. However, the **Admin** role seems to exist only for an organization, not for a personal account repository.


[Managing teams and people with access to your repository](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/managing-repository-settings/managing-teams-and-people-with-access-to-your-repository)

> For more information about repository roles, see "[Permission levels for a personal account repository](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-personal-account-settings/permission-levels-for-a-personal-account-repository)" and "[Repository roles for an organization](https://docs.github.com/en/organizations/managing-user-access-to-your-organizations-repositories/repository-roles-for-an-organization)."

**Comment by nsaunders** [comment]
Thanks @mechairoi, I think this will be good for now. I may either fork the repo or (if I'm not bothering you too much) follow up with a transfer request in the near future. There are some additional steps I need to complete before transferring, however. In any case, I really appreciate it!

So, with that, I'll go ahead and close this issue for now. Thanks again!

**Comment by mechairoi** [comment]
When you need to transfer, feel free to open another issue 😄
